.idea/misc.xml

```diff
@@ -39,7 +39,7 @@
       </value>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" project-jdk-name="1.8 (1)" project-jdk-type="JavaSDK">
```

Comment:
why was this changed?

Reply:
I think this is because I don't have a .gitignore. Maybe it will go back to the way it was run on your computer.

Reply:
I've brought this back to its original state